service-api/src/main/java/greencity/dto/RegionDto.java

```diff
@@ -1,14 +1,14 @@
 package greencity.dto;
 
-import lombok.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
-@Getter
-@Setter
+@Data
+@Builder
```

Comment:
Is it required to have setters and builders?

Reply:
done